common/log/log.go

```diff
@@ -118,7 +118,11 @@ func (l *Logger) SetDebugLevel(level int) error {
 func (l *Logger) output(level int, s string) error {
 	gid := GetGID()
 	gidStr := strconv.FormatUint(gid, 10)
-
+	//check the log filesize
+	if CheckIfNeedNewFile() {
+		ClosePrintLog()
```

Comment:
Is there any performance effect  after introduce this check for each  log print?